src/Microsoft.FeatureManagement/ConfigurationFeatureDefinitionProvider.cs

```diff
@@ -40,7 +41,7 @@ public void Dispose()
             _changeSubscription = null;
         }
 
-        public Task<FeatureDefinition> GetFeatureDefinitionAsync(string featureName)
+        public Task<FeatureDefinition> GetFeatureDefinitionAsync(string featureName, CancellationToken _)
```

Comment:
Maybe better to have a proper parameter name like `cancellationToken`

Reply:
I agree. I think the intent of using a discard here to indicate synchronous completion doesn't really fit will. I'll send a PR to update.